docs/catalog/cloud/evaluate-using-a-service-mesh.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- security
+- network
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Evaluate using a service mesh
+
+## Description
+A service mesh deploys additional containers for communication, typically in a sidecar pattern, to provide more operational capabilities leading to an increase in CPU usage and network traffic. 
+
+Nevertheless, it allows you to decouple your application from these capabilities as it moves them out from the application layer, and down to the infrastructure layer.
+
+## Solution
+Carefully consider the increase in CPU usage and network traffic generated by service mesh communication components before making the decision to use one.
+
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, not using a service mesh will impact:
+
+- `E`: By reducing CPU utilization and network traffic, we reduce the amount of energy required.
+- `M`: reducing CPU utilization and network data reduces the amount of resources required, which decreases the embodied carbon emissions.
+
+## Assumptions
+The application does not have compliance requirements for using a service mesh. 
+
+## Considerations
```

Comment:
I think some of this should be brought into the solution section. It's not clear based on the current content what an engineer should do in this scenario or what they should be considering. This section seems to highlight some of the actions or steps they can take into account.